### PR TITLE
Jfrog platform

### PR DESCRIPTION
**Pull request by MahithaB**
# Summary

Adding JFrog Platform quickstart with respective dashboards and install plan. 

## Pre merge checklist

- [ yes] Did you check you NRQL syntax? - Does it work?
- [ yes] Did you include an InstallPlan and Documentation reference?
- [ yes] Are all documentation links publically accessible?
- [yes ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)? 
- [ yes] Did you check your descriptive content for spelling and grammar errors?

### Dashboards 

- [yes ] Does the PR contain a screenshot for each of your dashboards?
- [ yes] Do your screenshots show data?
- [ yes] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts
No Alerts for our integration

### CODE REVIEW

File: quickstarts/jfrog-platform/config.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# quickstarts/example-category/example-quickstart/config.yml
+
+# This is automatically created by the system, this field should not be included by the author when creating a new quickstart.
+id:
```
Comment (aswanson-nr) [inline quickstarts/jfrog-platform/config.yml:4]: We can remove these lines, the system will automatically generate it for us!
```suggestion
```

File: quickstarts/jfrog-platform/config.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+# Relevant documentation for this quickstart
+documentation:
+  - name: JFrog Observability Solution
+    url: https://github.com/jfrog/log-analytics-newrelic
+    description: |
+      JFrog Github documentation
+
+
+# An icon or logo for the quickstart
+icon: icon.svg
```
Comment (aswanson-nr) [inline quickstarts/jfrog-platform/config.yml:50]: It looks like you're missing this `icon.svg` file. Could you add it into the pull request?

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Hey @MahithaB, thank you for your contribution! This is looking fantastic.
I noticed to small issues that need to be addressed, but then it should be good to go!

File: quickstarts/jfrog-platform/config.yml (hunk 1 above)
Comment (MahithaB) [reply quickstarts/jfrog-platform/config.yml]: Got it, removing id

File: quickstarts/jfrog-platform/config.yml (hunk 2 above)
Comment (MahithaB) [reply quickstarts/jfrog-platform/config.yml:45]: added icon.svg

**Comment by aswanson-nr** [comment]
@MahithaB This looks good to go to me! The PR is just out of date with the `main` branch on this repository, could you update it? Then I can hit the merge button!

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**